src/lotus/message/tests.rs

```diff
@@ -78,3 +83,41 @@ fn test_token_amount_from_str() {
     assert!(w.is_ok());
     assert_eq!(w.unwrap().token_amount, TokenAmount::from_whole(20));
 }
+
+#[test]
+fn test_subnet_info_to_str() {
+    let s = SubnetInfo {
+        id: Default::default(),
+        stake: Default::default(),
+        circ_supply: Default::default(),
+        status: Status::Active,
+    };
+
+    let w = serde_json::to_string(&s);
+    assert!(w.is_ok());
+}
+
+#[test]
+fn test_subnet_info_from_str() {
+    set_current_network(Network::Testnet);
+
+    let raw_str = r#"
+    {
+        "ID": {
+            "Parent": "/root",
+            "Actor": "t01002"
+        },
```

Comment:
We should check a grater value to see if it works with overflows (I had to fix a similar bug in deserialization  a few days ago

Reply:
yep, tested, it should be ok